### PR TITLE
Caesar Cipher using Python

### PR DESCRIPTION
**Pull request by photon149**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/smv1999/CompetitiveProgrammingQuestionBank/blob/master/CONTRIBUTING.md)?

Yes

### Description

I have implemented a Caesar Cipher substitution method using python language.In this we take input from the user whether he
wants to encrypt or decrypt.After that the user is asked to input a key value and input string, which then shows the results as per the user's selected choice.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

#640 

## Fill up the DCP form 

If you are a part of DCP 21, fill up this [form](https://docs.google.com/forms/d/e/1FAIpQLSeRAJnHE4eAa_W0IUeNZgheiy5POX3I3tQB7DtFsiB13D5w2g/viewform) after the PR is merged.

### The links provided in the form will be checked thoroughly, if someone tries fooling around, strict actions will be taken against the individual. 

##Additional Content
As per the conversation with Harsh on Slack , I have created a new folder named Cryptography.

### CODE REVIEW

**Review by smv1999 — CHANGES_REQUESTED** [review]
I am getting an indentation error while executing it and also the output is wrong for the following scenario.

CHOOSE FROM THE FOLLOWING : 
 1. Encrypt String 
 2. Decrypt String 
1
Enter the input text abc
Enter the key value 2
Encrypted string is : cdE

Correct Output : cde

**Comment by photon149** [comment]
Hey @smv1999 , sorry there was a little error from my side. I had forgot to mention a consideration while solving this problem .
The consideration is that , **Encrypted strings** consist of both upper and lowercase letters , while the decrypted strings/message only has **uppercase letters**.
I have made this update in my comments section of the file.
As for the indentation error , it's working fine for me . Since I m new to the open source working , can you recheck it once again or suggest some changes if it's not working correctly.

File: Cryptography/Caesar_Cipher.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    key_val = int(input('Enter the key value '))
+    print('Encrypted string is : ' + encrypt(input_string,key_val))
+    
+#This is the code for decryption choice , it calls the function and prints the decrypted value
+if (choose == 2):
+    encrypted_string = input('Enter the encrypted text ')
+    key_val = int(input('Enter the key value '))
+    print('Decrypted string is : ' + decrypt(encrypted_string,key_val))
+
+ """
```
Comment (noviicee) [inline Cryptography/Caesar_Cipher.py:60]: Fix the indentation and the multi-lines comment here. Use either single-quotes or double-quotes. Don't use both.

Comment (noviicee) [reply Cryptography/Caesar_Cipher.py:60]: Also, the program gives different outputs every time.

CHOOSE FROM THE FOLLOWING :

Encrypt String
Decrypt String
1
Enter the input text abc
Enter the key value 2
Encrypted string is : CDE
Correct Output : cde

CHOOSE FROM THE FOLLOWING :

Encrypt String
Decrypt String
1
Enter the input text abc
Enter the key value 2
Encrypted string is : cdE
Correct Output : cde

Kindly check once.

Comment (photon149) [reply Cryptography/Caesar_Cipher.py]: Oh okay got it , thanks fixed the indentation part .
Ugh ,, understood my error. Thanks for the help @noviicee corrected it.

**Review by noviicee — COMMENTED** [review]
Great job @photon149 But I think there is a slight error somewhere.
I am getting this output-
CHOOSE FROM THE FOLLOWING :
 1. Encrypt String
 2. Decrypt String
1
Enter the input text abc
Enter the key value 2
Encrypted string is : CDE

I think the correct answer should be cde.

**Comment by photon149** [comment]
Oh okay , I understood the problem. Can you recheck once . I have made the corrections.

**Comment by noviicee** [comment]
LGTM 👍

**Review by smv1999 — APPROVED** [review]

**Pull request merged**